client/state/reader/feeds/selectors.js

```diff
@@ -1,14 +1,29 @@
+const DAY_IN_MILLIS = 24 * 60 * 1000 * 1000;
+
 /**
 * Returns true if we should fetch the feed
 *
 * @param  {Object}  state  Global state tree
 * @param  {Number}  feedId The feed ID
 * @return {Boolean}        Whether feed should be fetched
 */
-
 export function shouldFeedBeFetched( state, feedId ) {
-	return ! state.reader.feeds.queuedRequests[ feedId ] && // not currently queued
-		! state.reader.feeds.items[ feedId ]; // not currently loaded
+	// we should fetch the feed if we don't have it,
+	// or we do have it
+	// and it's been more than a day since we last fetched, and it's not already queued to be fetched
+	return ! state.reader.feeds.queuedRequests[ feedId ] &&
+		(
+			! getFeed( state, feedId ) ||
+			! lastUpdatedWithin( state, feedId, DAY_IN_MILLIS )
+		);
+}
+
+function lastUpdatedWithin( state, feedId, timeInMillis ) {
```

Comment:
FFTI:
maybe make it return the flip of this with either something like isStale, isStalerThan, isOlderThan?